### PR TITLE
Vertex V1 / V2 Interop Support

### PR DESCRIPTION
**Pull request by frankfka**
@yruej301 can you advise on the new fields for V2?

### CODE REVIEW

File: packages/indexer-client/src/types/VertexTx.ts [hunk 1]
```diff
@@ -11,7 +11,10 @@ export interface VertexLiquidateSubaccountTx {
     sender: string;
     liquidatee: string;
     mode: number;
-    health_group: number;
+    // On V2 - should encode health group
+    product_id: number | undefined;
```
Comment (frankfka) [inline packages/indexer-client/src/types/VertexTx.ts:15]: Also - confirming that these are still u32 and coming back as numbers and not strings?

Comment (yruej301) [reply packages/indexer-client/src/types/VertexTx.ts:15]: will confirm!

Comment (yruej301) [reply packages/indexer-client/src/types/VertexTx.ts:15]: it's a number

Comment (frankfka) [reply packages/indexer-client/src/types/VertexTx.ts:15]: Okay this should be good for now then 👍🏻 but any other fields that should be added / removed?

**Pull request merged**